Fix example problems (#19258)
mdn/content: files/en-us/web/javascript/reference/operators/conditional_operator/index.md

        : condition3 ? value3
        : value4;
}
```

> Above is equivalent to below code.

Reviewer: ```suggestion
This can be re-written with an [`if...else`](/en-US/docs/Web/JavaScript/Reference/Statements/if...else) chain.
```

> This can be re-written with an [`if...else`](/en-US/docs/Web/JavaScript/Reference/Statements/if...else) chain.

Reviewer: ```suggestion
This can is equivalent to the following [`if...else`](/en-US/docs/Web/JavaScript/Reference/Statements/if...else) chain.
```

> This can is equivalent to the following [`if...else`](/en-US/docs/Web/JavaScript/Reference/Statements/if...else) chain.

Reviewer: ```suggestion
Thi is equivalent to the following [`if...else`](/en-US/docs/Web/JavaScript/Reference/Statements/if...else) chain.
```
